
From: David Howells <dhowells@redhat.com>

Here's a patch to erase the #error from linux/cachefs.h.  That was put in
there "temporarily" whilst I was working out that CONFIG_CACHEFS didn't get
set if CacheFS was made a module - CONFIG_CACHEFS_MODULE gets set instead.

Signed-Off-By: David Howells <dhowells@redhat.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/cachefs.h |    1 -
 1 files changed, 1 deletion(-)

diff -puN include/linux/cachefs.h~remove-error-from-linux-cachefsh include/linux/cachefs.h
--- 25/include/linux/cachefs.h~remove-error-from-linux-cachefsh	2004-09-08 13:09:40.989989056 -0700
+++ 25-akpm/include/linux/cachefs.h	2004-09-08 13:09:40.992988600 -0700
@@ -344,7 +344,6 @@ struct cachefs_page *cachefs_page_get_pr
 #ifdef CONFIG_CACHEFS
 	return __cachefs_page_get_private(page, gfp);
 #else
-#error
 	return ERR_PTR(-EIO);
 #endif
 }
_
